tdrs-frontend/src/selectors/auth.js

```diff
@@ -6,6 +6,25 @@ export const selectUser = (state) => state.auth.user || null
 export const selectUserAccountApprovalStatus = (state) =>
   selectUser(state)?.['account_approval_status']
 
+export const selectUserRoles = (state) => selectUser(state)?.['roles'] || []
```

Comment:
this is (hopefully) an improvement to avoid us getting in similar situations in the future. We should opt to standardize the selectors as much as possible in order to ensure the same implementation across the app.